src/DIRAC/Resources/Storage/EchoStorage.py

```diff
@@ -160,7 +163,11 @@ def _removeSingleFile(self, path):
         """
 
         startTime = default_timer()
-        res = super(EchoStorage, self)._removeSingleFile(path)
+        with setGfalSettings(self.ctx, "CORE", "NAMESPACE_TIMEOUT", REMOVAL_TIMEOUT):
```

Comment:
This should also work:
```python
with (setGfalSettings(...),
      setGfalSettings(...)):
   pass
```

Reply:
Only in Python 3.10+: https://docs.python.org/3.10/whatsnew/3.10.html#parenthesized-context-managers

(3.9 optionally has this feature but is targetting v7r3 and needs to work with Python 2.7)

Reply:
weird, this is claimed as a python3.10 feature, but seems to work in python 3.9.12 as well
```python
with (open('/dev/null') as foo,
      open('/dev/zero') as bar):
      print("baz!")
```
But not python 2.7, so 
```python
with open('/dev/null') as foo, \
     open('/dev/zero') as bar:
      print("baz!")
```

Reply:
In Python 3.9 it's a build flag/environment variable to control it. I don't know how widely enabled it is outside of conda-forge.

```python
$ ipython
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:33)
Type 'copyright', 'credits' or 'license' for more information
IPython 8.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import tempfile
   ...: with (tempfile.NamedTemporaryFile(), tempfile.NamedTemporaryFile()):
   ...:     pass
   ...:
   
$ PYTHONOLDPARSER=x ipython
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:33)
Type 'copyright', 'credits' or 'license' for more information
IPython 8.4.0 -- An enhanced Interactive Python. Type '?' for help.
In [1]: import tempfile
   ...: with (tempfile.NamedTemporaryFile(), tempfile.NamedTemporaryFile()):
   ...:     pass
   ...:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [1], in <cell line: 2>()
      1 import tempfile
----> 2 with (tempfile.NamedTemporaryFile(), tempfile.NamedTemporaryFile()):
      3     pass

AttributeError: __enter__
```

Reply:
Apprently it can be expected to be enabled everywhere as black will reformat to modernise the `\` automatically if it's targetting Python 3.9: https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html#using-backslashes-for-with-statements